src/main/java/gregtech/common/metatileentities/storage/CachedRecipeData.java

```diff
@@ -111,7 +108,7 @@ public boolean getIngredientEquivalent(int slot) {
                     }
                 }
                 if (!matched) {
-                    map.put(itemStackKey, false);
+                    map.put(itemStack, false);
```

Comment:
Should this be a copy? Considering there was a deleted call to `ItemStackKey#getItemStack`